.github/workflows/tests.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Install dependencies
         id: install-dependencies
         run: |
-          pip install .[tests,minver]
+          pip install .[tests,visualisation,minver]
```

Comment:
We use 'requests' library for fetching data from mermaid.ink (I couldn't figure out how to do this without requests right now).